### PR TITLE
Enhance typo3 global command with options

### PR DESCRIPTION
**Pull request by gilbertsoft**
## The Problem/Issue/Bug:

This patch adds the options -f|--first-install and -e|--enable-install-tool to create ${DOCROOT}/FIRST_INSTALL respectively ${DOCROOT}/typo3conf/ENABLE_INSTALL_TOOL which are often used to setup and configure a TYPO3 project.

The output of `ddev help typo3` is now:

```
Run TYPO3 CLI (typo3) command inside the web container (global shell web container command)

Usage:
  ddev typo3 [flags] [command] [options] [arguments]

Examples:
  ddev typo3 --first-install
  ddev typo3 list
  ddev typo3 site:list
  ddev typo3 extension:list

Flags:
  -e, --enable-install-tool   Enable the TYPO3 Install Tool by creating the file $DOCROOT/typo3conf/ENABLE_INSTALL_TOOL
  -f, --first-install         Enable the TYPO3 Install Wizard by creating the file $DOCROOT/FIRST_INSTALL
  -h, --help                  help for typo3

Global Flags:
  -j, --json-output   If true, user-oriented output will be in JSON format.
```

## Manual Testing Instructions:
- run `ddev typo3 -f` or `ddev typo3 --first-install` and check if ${DOCROOT}/FIRST_INSTALL is created
- run `ddev typo3 -e` or `ddev typo3 --enable-install-tool` and check if ${DOCROOT}/typo3conf/ENABLE_INSTALL_TOOL is created

### CODE REVIEW

File: cmd/ddev/cmd/global_dotddev_assets/commands/web/typo3 [hunk 1]
```diff
@@ -4,8 +4,30 @@
...
 
+function create_file {
+    touch $1
+    printf '"%s" was created successfully.\n' "$1" >&2
+}
+
+while :; do
+    case ${1:-} in
+        -f|--first-install)
+            create_file ${DOCROOT}/FIRST_INSTALL
```
Comment (gilbertsoft) [inline cmd/ddev/cmd/global_dotddev_assets/commands/web/typo3:19]: needs to be changed to DDEV_DOCROOT

File: cmd/ddev/cmd/global_dotddev_assets/commands/web/typo3 [hunk 2]
```diff
@@ -4,8 +4,30 @@
...
+            exit
+            ;;
+        -e|--enable-install-tool)
+            create_file ${DOCROOT}/typo3conf/ENABLE_INSTALL_TOOL
```
Comment (gilbertsoft) [inline cmd/ddev/cmd/global_dotddev_assets/commands/web/typo3:23]: needs to be changed to DDEV_DOCROOT

File: cmd/ddev/cmd/global_dotddev_assets/commands/web/typo3 [hunk 3]
```diff
@@ -4,8 +4,30 @@
 # This assumes that the typo3 command will be in the $PATH; if in vendor/bin/ it will be
 
 ## Description: Run TYPO3 CLI (typo3) command inside the web container
-## Usage: typo3 [args]
-## Example: "ddev typo3 site:list" or "ddev typo3 list" or "ddev typo3 extension:list"
+## Usage: typo3 [-f|--first-install] [-e|--enable-install-tool] [flags] [command] [options] [arguments]
+## Example: "ddev typo3 --first-install" or "ddev typo3 list" or "ddev typo3 site:list" or "ddev typo3 extension:list"
 ## ProjectTypes: typo3
```
Comment (gilbertsoft) [inline cmd/ddev/cmd/global_dotddev_assets/commands/web/typo3:9]: Add Flags definition

**Comment by gilbertsoft** [comment]
#2493 needs to be merged first to allow a flags definition

**Comment by rfay** [comment]
I'm going to close this for now, but really happy to open it up again, just a comment will do it. But it's quite stale and I know you have other things going. I sure welcome you back to it when you get time.

**Pull request closed without merging**